overlord/hookstate/hookmgr.go

```diff
@@ -216,8 +218,13 @@ func (m *HookManager) doRunHook(task *state.Task, tomb *tomb.Tomb) error {
 
 	if hookExists {
 		output, err := runHook(context, tomb)
-		if err != nil {
+		if hooksup.IgnoreFail && err != nil {
+			task.State().Lock()
+			task.Logf("ignoring failure in hook %q: %v", hooksup.Hook, err)
+			task.State().Unlock()
+		} else if err != nil {
 			err = osutil.OutputErr(output, err)
```

Comment:
sorry, I mean do this before the if ? it seems we are not logging the output if ignore the failure